modin/pandas/groupby.py

```diff
@@ -41,8 +41,43 @@
 from .utils import is_label
 
 
+_DEFAULT_BEHAVIOUR = {
```

Comment:
On what basis was this list determined?

Reply:
This just contains a set of private methods that should always operate based on Modin's DataFrameGroupBy object state. We don't want these methods to fall back to pandas so they wouldn't return unexpected results in case of private method names collision.